components/board/mcp3008.go

```diff
@@ -31,7 +31,9 @@ func (mar *MCP3008AnalogReader) Read(ctx context.Context, extra map[string]inter
 	if err != nil {
 		return 0, err
 	}
-	val := (int(rx[1]) << 8) | int(rx[2]) // reassemble 10 bit value
+	// Reassemble the 10-bit value. Do not include bits beyond the final 10, because they contain
```

Comment:
"Beyond" implies "past the end," but we really want to exclude bits _before_ the final 10. Rephrase this.